README.md

```diff
@@ -14,9 +14,9 @@ The required keychain items should also have been allowed access for the relevan
 
 ### Features
 
-- Signs binaries, dmgs, pkgs, or apps for MacOS.
+- Signs binaries, dmgs, or apps for MacOS.
 - Notarization, with stapling of the notarization ticket.
-- One or more targets can be specified for signing at once.
+- Specifying multiple sub targets to sign. For example: the frameworks in a `.app`.
```

Comment:
So the user needs to specify these, and we don't try to figure them out on our own?

I'm not 100% sure what this would look like in practice. It might make sense to include an example of this use case in the examples section below.

Reply:
Yep, everything has to be manually specified.
And its not a pleasant experience either; the codesigning tends to work fine even if the sub-assets aren't signed. But then the notarisation fails (which can take "minutes to hours") and reports _some_ of the unsigned issues. You fix, you repeat, get the next issues... Not a great iteration loop.

Would absolutely _love_ to be able to simplify this, open to suggestions if you have any.